### ISSUE TITLE
RPLIDAR-A1 Drifting in ROS 2 

### ISSUE THREAD

**Issue by QQting** [allenh1/rplidar_ros#15]
Hi there,

I found my RPLidar-A1 drift much more in ROS 2 than ROS 1.
Does anyone encounter this issue?

**ROS 1 - Melodic**
![rplidar_drift_ros1](https://user-images.githubusercontent.com/320113/87402127-b1da0d80-c5ed-11ea-9307-b34af54086d2.gif)

**ROS 2 - Foxy**
![rplidar_drift_ros2](https://user-images.githubusercontent.com/320113/87402204-ccac8200-c5ed-11ea-8b70-6bad2174aaa4.gif)

**Comment by QQting** [comment]
@allenh1 , Do you have any ideas about this?

**Comment by allenh1** [comment]
@QQting no, but I plan to look into this shortly. Do you know if the drift also occurs in the non composition node? (This would be the legacy exe).

**Comment by QQting** [comment]
@allenh1 , yes. it also occurs in non composition rplidarNode.
